README.md

```diff
@@ -1,11 +1,28 @@
-vtkAddon
-========
+# vtkAddon
 
-Overview
---------
+## Overview
 
 VTK C++ functions and filters for medical imaging applications.
```

Comment:
Add something like this here: This module contains general-purpose features that may be integrated into VTK library in the future.